---
layout: archive
title: "Projects"
---


### Software

Our NGS analysis relies on the [bcbio.nextgen](https://github.com/chapmanb/bcbio-nextgen) framework, a community-developed NGS workflow that comes with [full documentation](https://bcbio-nextgen.readthedocs.org/en/latest/), is open source under the [MIT license](http://opensource.org/licenses/mit-license.html), and is in use at over a dozen sites internationally. It is installed at both the FAS and HMS Research Computing environments and provides researchers with best practice workflows for [exome / whole-genome sequencing](https://bcbio-nextgen.readthedocs.org/en/latest/contents/pipelines.html#variant-calling) (built around GATK 3.0 and FreeBayes) and [RNA-Seq](https://bcbio-nextgen.readthedocs.org/en/latest/contents/pipelines.html#rna-seq) (TopHat2/STAR, edgeR/DESeq/limma). Take a look at some of the key blog posts around bcbio:

* [Scaling NGS pipelines for WGS](https://bcbio.wordpress.com/2013/05/22/scaling-variant-detection-pipelines-for-whole-genome-sequencing-analysis/)
* [Ensembl variant calling](https://bcbio.wordpress.com/2013/10/21/updated-comparison-of-variant-detection-methods-ensemble-freebayes-and-minimal-bam-preparation-pipelines/)
* [Docker for easier distribution](https://bcbio.wordpress.com/)
* [Introducing RNA-Seq to bcbio](http://spliced.ghost.io/automated-comparison-of-rna-seq-differential-expression-calls/)


### Recent presentations

* [Clinical Sequencing](https://dl.dropboxusercontent.com/u/407047/Work/Presentations/20140503%20Glasgow%20Clinical%20Sequencing.pdf) (University of Glasgow)
* [bcbio implementation details](https://github.com/chapmanb/bcbb/raw/master/talks/abcd2014_bcbio_nextgen/chapman_bcbio.pdf) (Harvard ABCD Group)
* [Workflows for clinical sequencing](https://dl.dropboxusercontent.com/u/407047/Work/Presentations/20140218%20Sheffield%20NGS%20in%20Public%20Health.pdf) (Sheffield Institute for
Translational Neuroscience)
* [A framework for RNA-Seq DE comparisons](https://dl.dropboxusercontent.com/u/407047/Work/Presentations/20140200%20BIG-meeting-feb-2014.pdf) (BIG Meeting Group)
* [Community-developed NGS pipelines](https://dl.dropboxusercontent.com/u/407047/Work/Presentations/20131102%20CSHL%20Genome%20Informatics/20131101%20CSHL%20GI2013%20bcbio.pdf) (2013 Genome Informatics Meeting, CSHL)


### Selected publications

* FANTOM Consortium and the RIKEN PMI and CLST (DGT). (2014). _A promoter-level mammalian expression atlas._ Nature, 507(7493), 462–470. ([Pubmed](http://www.ncbi.nlm.nih.gov/pubmed/24670764))
* Zook, J. M., Chapman, B., Wang, J., Mittelman, D., Hofmann, O., Hide, W., & Salit, M. (2014). _Integrating human sequence data sets provides a resource of benchmark SNP and indel genotype calls._ Nature Biotechnology. ([Pubmed](http://www.ncbi.nlm.nih.gov/pubmed/24531798))
* Liu, J., Krautzberger, A. M., Sui, S. H., Hofmann, O. M., Chen, Y., Baetscher, M., et al. (2014). _Cell-specific translational profiling in acute kidney injury. Journal of Clinical Investigation_, 124(3), 1242–1254. ([Pubmed](http://www.ncbi.nlm.nih.gov/pubmed/24569379))
* Huang, H.-T., Kathrein, K. L., Barton, A., Gitlin, Z., Huang, Y.-H., Ward, T. P., et al. (2013). _A network of epigenetic regulators guides developmental haematopoiesis in vivo._ Nature Cell Biology, 15(12), 1516–1525. ([Pubmed](http://www.ncbi.nlm.nih.gov/pubmed/24240475))
* Altschuler, G. M., Hofmann, O., Kalatskaya, I., Payne, R., Ho Sui, S. J., Saxena, U., et al. (2013). _Pathprinting: An integrative approach to understand the functional basis of disease._ Genome Medicine, 5(7), 68. ([Pubmed](http://www.ncbi.nlm.nih.gov/pubmed/23890051))
* Sui, S. H., Merrill, E., Gehlenborg, N., Haseley, P., Sytchev, I., Park, R., et al. (2013). _The Stem Cell Commons: an exemplar for data integration in the biomedical domain driven by the ISA framework._ AMIA Summits on Translational Science Proceedings, 2013, 70–70. ([Pubmed](http://www.ncbi.nlm.nih.gov/pubmed/24303302))
* Paila, U., Chapman, B. A., Kirchner, R., & Quinlan, A. R. (2013). _GEMINI: integrative exploration of genetic variation and genome annotations._ PLoS Computational Biology, 9(7). ([Pubmed](http://www.ncbi.nlm.nih.gov/pubmed/23874191))




