---
layout: archive
title: "Training"
---

With funding from the HMS Tools and Technology Committee, the Harvard NeuroDiscovery Center (HNDC) and the Harvard Stem Cell Institute (HSCI) we have a dedicated bioinformatics training program for:

- HMS researchers with a primary affiliation in one of the Basic and Social Science Departments on the Quad 

- researchers with an affiliation with the HNDC

- HSCI researchers 

---

### Training program description:

We provide bioinformatics training through both **shorter workshops** and **in-depth courses**. [[Click here](http://bioinformatics.sph.harvard.edu/training/#upcoming-workshopscourses) for a list of upcoming workshops/courses, or scroll down.]

#### 1. Workshops 

* **Next Generation Sequencing Introduction Series**

	([Galaxy-based](https://wiki.galaxyproject.org/)) workshops on RNA-Seq, ChIP-Seq to introduce basic concepts of Next-Generation Sequencing (NGS) analysis. The goal of these workshops are to enable researchers to design their studies appropriately and perform preliminary data analyses.

* **Basic Bioinformatics Skills Series**	

	Topics include R, data visualization using R, Python programming, Unix and high-performance computing (duration varies from half a day to three days).

#### 2.  In-Depth Next Generation Sequencing Analysis Courses

These intensive courses run for 10-12 days and are aimed at bench biologists interested in learning **how to perform independent, best practice NGS-based analyses**. Topics include:

- Unix & High-Performance Computing ([Orchestra](https://rc.hms.harvard.edu/#orchestra))
- NGS data analysis (RNA-Seq, ChIP-Seq, Variant calling)
- Statistical analysis using R
- Functional analysis


*No prior NGS or command line expertise is required for our workshops or courses unless explicitly stated.*

---

### Upcoming workshops/courses:

#### For HSCI researchers only

* **January 25th and 26th, 2017: *Introduction to R***

	This 2-day hands-on workshop will introduce participants to using R and RStudio. After this workshop participants can expect to be comfortable with the basics of data manipulation and plotting using R. 
	
	** *Registration will open on January 4th, 2017* **
	
* **February 2nd, 2017: *Introduction to Differential Gene Expression analysis***

	This 1-day hands-on workshop is designed to teach researchers the basic concepts for performing differential gene expression analysis using tools in R. 
	
	**Prerequisite: *Introduction to R*** (You are eligibile to attend only if you have taken an R course with us prior to this workshop.)
	
	** *If you have R expertise and would like to attend this workshop please [email us](mailto:hbctraining@hsph.harvard.edu) with the subject "February DGE Workshop Inquiry".* **
		
* **March 8th and 9th, 2017: *Introduction to UNIX and RNA-Seq***
  
	This 2-day hands-on workshop will introduce participants to the command-line interface, compute clusters, and basic methods to get started with RNA-Seq data analysis.
	
	** *Registration will open on January 23rd, 2017!* **

#### For HMS (on-quad) and HNDC researchers only

* **February 15th and 16th, 2017: *Introduction to UNIX and RNA-Seq***
  
	This 2-day hands-on workshop will introduce participants to the command-line interface, compute clusters, and basic methods to get started with RNA-Seq data analysis.
	
	** *Registration will open on February 13th, 2017!* **
	
* **Spring 2017 workshops schedule will be announced soon!**

---

### Contact:

For any training-related questions, please email us at [hbctraining@hsph.harvard.edu](mailto:hbctraining@hsph.harvard.edu).


